src/main/java/com/image/UrlImage.java

```diff
@@ -0,0 +1,32 @@
+package com.image;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class UrlImage implements Image{
+
+    URL url;
+    User user;
+
+    public void setUser(User user) {
```

Comment:
@PavelNejinskiy please no setters, the intension is to create immutable objects if we can.